web/packages/teleport/src/Discover/Database/DeployService/ManualDeploy/ManualDeploy.story.tsx

```diff
@@ -0,0 +1,123 @@
+/**
+ * Copyright 2022 Gravitational, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import { MemoryRouter } from 'react-router';
+
+import { Context as TeleportContext, ContextProvider } from 'teleport';
+import cfg from 'teleport/config';
+import { ResourceKind } from 'teleport/Discover/Shared';
+import { PingTeleportProvider } from 'teleport/Discover/Shared/PingTeleportContext';
+import { getUserContext } from 'teleport/mocks/contexts';
+import { FeaturesContextProvider } from 'teleport/FeaturesContext';
+import {
+  DatabaseEngine,
+  DatabaseLocation,
+} from 'teleport/Discover/SelectResource';
+import {
+  DiscoverProvider,
+  DiscoverContextState,
+} from 'teleport/Discover/useDiscover';
+
+import ManualDeploy from './ManualDeploy';
+
+export default {
+  title: 'Teleport/Discover/Database/Deploy/Manual',
+};
+
+export const Init = () => {
+  return (
+    <Provider>
+      <ManualDeploy />
+    </Provider>
+  );
+};
+
+export const InitWithLabels = () => {
+  return (
+    <Provider
+      agentMeta={{
+        agentMatcherLabels: [
+          { name: 'env', value: 'staging' },
+          { name: 'os', value: 'windows' },
+        ],
+      }}
+    >
+      <ManualDeploy />
+    </Provider>
+  );
+};
+
+const Provider = props => {
+  const ctx = createTeleportContext();
+  const discoverCtx: DiscoverContextState = {
+    agentMeta: {
+      resourceName: 'db-name',
+      agentMatcherLabels: [],
+      db: {} as any,
+      selectedAwsRdsDb: {} as any,
+      ...props.agentMeta,
+    },
+    currentStep: 0,
+    nextStep: () => null,
+    prevStep: () => null,
+    onSelectResource: () => null,
+    resourceSpec: {
+      dbMeta: {
+        location: DatabaseLocation.Aws,
+        engine: DatabaseEngine.AuroraMysql,
+      },
+    } as any,
+    exitFlow: () => null,
+    viewConfig: null,
+    indexedViews: [],
+    setResourceSpec: () => null,
+    updateAgentMeta: () => null,
+    emitErrorEvent: () => null,
+    emitEvent: () => null,
+    eventState: null,
+  };
+
+  return (
+    <MemoryRouter
+      initialEntries={[
+        { pathname: cfg.routes.discover, state: { entity: 'database' } },
+      ]}
+    >
+      <ContextProvider ctx={ctx}>
+        <FeaturesContextProvider value={[]}>
+          <DiscoverProvider mockCtx={discoverCtx}>
+            <PingTeleportProvider
+              interval={props.interval || 100000}
```

Comment:
Thoughts on moving `100000` to a constant? `const DEFAULT_PING_INTERVAL = 1000 * 100; // 100 seconds`